app/lib/filter_query_builder.rb

```diff
@@ -23,6 +23,7 @@ def build_filter(facet)
     filter_class = {
       'date' => DateFilter,
       'text' => TextFilter,
+      'topical' => TopicalFilter,
```

Comment:
https://github.com/alphagov/finder-frontend/blob/master/docs/finder-content-item.md#type needs updating for this I think.